### DISCUSSION TITLE
Has anyone done something like this for the 8088?

### DISCUSSION THREAD

**Discussion by BOBBYWY** [wwarthen/RomWBW#286] [General]
Just asking.  I am not even sure it would be possible.  

It seems most of the parts are available under CP\M-86,  But the packaging is one major piece I can't grasp.

**Comment by wwarthen** [comment]
As far as I know, there is nothing quite like RomWBW for CP/M-86.

I guess it would be possible.  The IBM-PC standard is so predominant in the 8086 world that I'm not entirely sure how useful it would be.  I guess it would be nice for hobbyists constructing 8086-based systems.  Is there a specific platform you are interested in?

**Reply by BOBBYWY** [reply]
Thanks for the reply.  Looks like I will need to dig in and study a lot.  CP\M-86 looks like it would be possible to adapt, but I am not tied to it.  Maybe a RTOS would work better for me.  As long as I don't start dreaming in Hex and binary, like I did in university using keypunch machines for programming.

While I was shopping eBay I picked up a few N80C188 chips, and now I am trying to figure out what to do with them.  Some incompatibility with the 8088 standards.  I know about the retrobrew sbc-188, I would want to trim off the floppy and hard drive capability and use compact flash or SD cards.  

USB C for power and communications.   TI MSP430 chips for glue and interfacing.  Terminal only.  Vague mental images so far.
